src/FakeItEasy/ReturnValueConfigurationExtensions.cs

```diff
@@ -131,7 +131,7 @@ internal static void ReturnsNextFromQueue<T>(this IReturnValueConfiguration<T> c
             configuration.ReturnsLazily(x =>
             {
                 queue.TryDequeue(out T returnValue);
-                return returnValue;
+                return returnValue!;

```

Comment:
This can't actually be null. TryDequeue will always return true, since we execute this only `queue.Count` times.
(and the ConcurrentQueue isn't exposed publicly, so it can't be modified somewhere else)